packages/capacitor-plugin/package.json

```
@@ -0,0 +1,89 @@
+{
+  "name": "@capacitor/filesystem",
+  "version": "7.0.0",
+  "description": "Filesystem plugin for Capacitor",
```

Minor: To avoid changing the description on NPM of the current plugin (unless we want to due to API changes), we should have put the existing one:

```suggestion
  "description": "The Filesystem API provides a NodeJS-like API for working with files on the device.",
```